packages/manager/apps/carrier-sip/src/endpoints/routing.js

```diff
@@ -3,6 +3,10 @@ export default /* @ngInject */ ($stateProvider) => {
     url: '/endpoints',
     component: 'carrierSipEndpoints',
     resolve: {
+      endpointIpList: /* @ngInject */ (endpoints) =>
```

Comment:
I'm a bit confused to see that we duplicate resolve code, but seems to be legit as the way we develop `carrier-sip` module (don't embed the routing).

Reply:
Yes good point, I'll check if I can set the routing directly in the module. (In another PR)

Reply:
Maybe the module could expose the service to avoid duplicating code

Reply:
@cbourgois / @marie-j a task has been created to address your point. (MANAGER-5427)